[2024-09-19T23:22:04.320+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-19T23:22:04.687+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_postgres_pandas_dag_with_decorators.process_data manual__2024-09-19T23:21:41.684231+00:00 [queued]>
[2024-09-19T23:22:04.745+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_postgres_pandas_dag_with_decorators.process_data manual__2024-09-19T23:21:41.684231+00:00 [queued]>
[2024-09-19T23:22:04.746+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-09-19T23:22:04.799+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): process_data> on 2024-09-19 23:21:41.684231+00:00
[2024-09-19T23:22:04.853+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=508) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-19T23:22:04.856+0000] {standard_task_runner.py:63} INFO - Started process 510 to run task
[2024-09-19T23:22:04.845+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'example_postgres_pandas_dag_with_decorators', 'process_data', 'manual__2024-09-19T23:21:41.684231+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/etl_prueba.py', '--cfg-path', '/tmp/tmpnlnk_kzt']
[2024-09-19T23:22:04.865+0000] {standard_task_runner.py:91} INFO - Job 67: Subtask process_data
[2024-09-19T23:22:05.149+0000] {task_command.py:426} INFO - Running <TaskInstance: example_postgres_pandas_dag_with_decorators.process_data manual__2024-09-19T23:21:41.684231+00:00 [running]> on host 5e909a8df6c4
[2024-09-19T23:22:06.488+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_postgres_pandas_dag_with_decorators' AIRFLOW_CTX_TASK_ID='process_data' AIRFLOW_CTX_EXECUTION_DATE='2024-09-19T23:21:41.684231+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-19T23:21:41.684231+00:00'
[2024-09-19T23:22:06.490+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-19T23:22:06.515+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2024-09-19T23:22:06.559+0000] {logging_mixin.py:188} WARNING - /opt/***/dags/etl_prueba.py:58 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2024-09-19T23:22:06.580+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-09-19T23:22:06.581+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-19T23:22:06.607+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=example_postgres_pandas_dag_with_decorators, task_id=process_data, run_id=manual__2024-09-19T23:21:41.684231+00:00, execution_date=20240919T232141, start_date=20240919T232204, end_date=20240919T232206
[2024-09-19T23:22:06.700+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-09-19T23:22:06.737+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-19T23:22:06.749+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
